src/styles/style.css

```diff
@@ -0,0 +1,84 @@
+@import url(./stars.css);
+
+html,
+h2,
+p {
+  margin: 0;
+  font-family: Roboto, sans-serif;
```

Comment:
add `font-family` only for html

Reply:
I thought I can add font-family, and remove margin in sametime, sorry for this) it's my bad